image_classification/reader.py

```diff
@@ -10,7 +10,10 @@ def train_mapper(sample):
     '''
     img, label = sample
     img = paddle.image.load_image(img)
-    img = paddle.image.simple_transform(img, 256, 224, True)
+    # https://github.com/PaddlePaddle/Paddle/blob/d52fa26fdab7a0497a3e7f49833d1b3827955c44/python/paddle/v2/dataset/flowers.py#L65
+    # mean:根据加载的预训练模型的mean设置。如果不加载预训练模型，则根据自己喜好设置
+    # paddlepaddle的代码中，默认减去这个mean值，如果加载的预训练模型使用的mean与这个值不一样，需要修改
```

Comment:
Thanks for your contributions. Could you change the comments to English or remove these comments?

Reply:
I had  changed the comments in English.